main/acle.md

```diff
@@ -2756,9 +2757,7 @@ The following table lists the architectures feature mapping for AArch64
    | 490           | `FEAT_SSBS`              | ssbs          | ```ID_AA64PFR1_EL1.SSBS >= 0b0001```      |
    | 500           | `FEAT_SSBS2`             | ssbs2         | ```ID_AA64PFR1_EL1.SSBS >= 0b0010```      |
    | 510           | `FEAT_BTI`               | bti           | ```ID_AA64PFR1_EL1.BT >= 0b0001```        |
-   | 520           | `FEAT_LS64`              | ls64          | ```ID_AA64ISAR1_EL1.LS64 >= 0b0001```     |
-   | 530           | `FEAT_LS64_V`            | ls64_v        | ```ID_AA64ISAR1_EL1.LS64 >= 0b0010```     |
-   | 540           | `FEAT_LS64_ACCDATA`      | ls64_accdata  | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011```     |
+   | 520           | `FEAT_LS64`,`FEAT_LS64_V`,<br>`FEAT_LS64_ACCDATA`| ls64 | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011``` |
```

Comment:
```suggestion
   | 520           | `FEAT_LS64`,`FEAT_LS64_V`,<br>`FEAT_LS64_ACCDATA`| ls64_accdata | ```ID_AA64ISAR1_EL1.LS64 >= 0b0011``` |
```
I'm not sure which way to go here but it's worth discussing. Generally FMV keeps the names of the features matching with the architectural `FEAT_` names, and the current PR makes `ls64` inconsistent with `FEAT_LS64_ACCDATA`. If we ever did want to add independent `FEAT_LS64` and `FEAT_LS64_V` to FMV, we would have the same problem we have in the compilers where `ls64` can't be reassigned from `FEAT_LS64_ACCDATA` to `FEAT_LS64`. However I can see that it makes the feature name inconsistent with the existing compiler command line spellings.

Reply:
Being inconsistent with the command line flags could be mitigated with a warning like `no such feature ls64; did you mean ls64_accdata?`

Reply:
I think we want to stay aligned with the command line/assembler directive here, otherwise we will create further confusion. But that's my opinion. @andrewcarlotti, any opinion?

Reply:
I agree that FMV feature names should match command line and assembler directive names.